gcsfs/core.py

```diff
@@ -893,17 +897,33 @@ def flush(self, force=False):
             raise ValueError('Flush on a file not in write mode')
         if self.closed:
             raise ValueError('Flush on closed file')
+
         if self.buffer.tell() == 0 and not force:
             # no data in the buffer to write
             return
+        if self.buffer.tell() < GCS_MIN_BLOCK_SIZE and not force:
+            warnings.warn(
+                "GCSFile.flush(force=False) with buffer size (%s) below minimum GCS chunk size (2 ** 18), "
```

Comment:
I'm not certain I want this warning showing up just  because the user has used text-mode on a gcsfs output. Since the operation doesn't actually do anything, but the data is sure to get written eventually, I would be happy to have this silent.

Reply:
I definitely struggled with this. It seems strange to not warn in the case of an ignored-flush, but I suppose that non-forced flushes aren't terribly common. Dropping to `DEBUG` level logging sound reasonable?

Reply:
Yes, that sounds reasonable.